src/services/users.service.js

```diff
@@ -0,0 +1,59 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { User } from './../models/index.js';
+import { v4 as uuidv4 } from 'uuid';
+import { emailService } from './index.js';
+
+export const usersService = {
+  async register({ email, password, name }) {
+    const isExist = !!(await this.getOne({ email }));
+
+    if (isExist) {
+      throw ApiError.badRequest('User already exist', {
+        email: 'User already exist',
+      });
+    }
+
+    const activationToken = uuidv4();
+
+    await User.create({
+      email,
+      password,
```

Comment:
Ensure that the `password` is hashed before being stored in the database. Storing plain text passwords is a security risk.